apps/communications/contacts/js/views/list.js

```diff
@@ -1715,9 +1715,8 @@ contacts.List = (function() {
         callback(uuid);
       });
     }
-    }
-    evt.preventDefault();
```

Comment:
is this line really needs to be deleted, I'm just wondering because I discovered, that in initial patch it exists.